sass/tji/_variables.scss

```diff
@@ -17,16 +18,26 @@ $tji-spacing-lg: 2rem;
 $tji-spacing-xl: 3rem;
 
 /* typography */
-$tji-font-size-xs: 1.4rem;
-$tji-font-size-sm: 1.7rem;
-$tji-font-size-md: 2.6rem;
-$tji-font-size-lg: 3.6rem;
-$tji-font-size-xl: 4.4rem;
+$tji-font-size-xs: 0.8rem;
+$tji-font-size-sm: 1.4rem;
+$tji-font-size-md: 1.7rem;
+$tji-font-size-lg: 2.6rem;
+$tji-font-size-xl: 3.6rem;
 
-$tji-line-height-xs: 2.4rem;
-$tji-line-height-sm: 2.7rem;
-$tji-line-height-md: 3.6rem;
-$tji-line-height-lg: 4.6rem;
-$tji-line-height-xl: 5.4rem;
+$tji-line-height-xs: 1.0rem;
+$tji-line-height-sm: 2.4rem;
+$tji-line-height-md: 2.7rem;
+$tji-line-height-lg: 3.6rem;
+$tji-line-height-xl: 4.6rem;
 
 $tji-chart-legend-size: 1rem;
+
+/* dividers */
+$tji-border: 1px solid #ccc;
+
+/* break-points */
+$tji-break-point-xs: 37.5rem;
```

Comment:
having our base font-size be 16px (instead of 10px) makes any em/rem based calculation really complicated. Maybe when we have extra time at some point we want to go back an adjust our base to 10px to make these numbers more intuitive.

Reply:
Thanks Simi, I had not used flexbox grid before and while searching for the breakpoints found an extended version of flexbox grid for sass, which apparently has slightly different breakpoints. I will adjust these to match the original flexbox grid.

Agreed on adjusting the base font-size. I think in general we need to go through following this and improve some practices and consistency.